pkg/render/node_test.go

```diff
@@ -76,8 +76,9 @@ var _ = Describe("Node rendering tests", func() {
 			},
 			CalicoNetwork: &operatorv1.CalicoNetworkSpec{
 				BGP:                        &bgpEnabled,
-				IPPools:                    []operatorv1.IPPool{{CIDR: "192.168.1.0/16"}},
```

Comment:
I think we should keep the general default tests to be IPv4 only. I think we should add some additional tests that include an IPv6 pool and include mtu setting in those tests also, those new tests should only test for the specific differences we expect (in other words to include checking every Env var and setting that we do in the default tests).

Reply:
Yeah, I think having separate v4, v6, and dual-stack tests would be the right move

Reply:
Agreed, will do that! Thanks!

Reply:
Just force-pushed a commit extending the whole node_test.go to run for IPv4, IPv6 and dual-stack. Please let me know your thoughts. Thanks!